### ISSUE TITLE
reveal.js seems empty

### ISSUE THREAD

**Issue by kohlhase** [OpenDreamKit/OpenDreamKit.github.io#55]
I am trying to do the slides and run jekyll locally. I am getting lots of errors (and there is no styling in the slides): 
```
[2017-04-19 07:52:59] ERROR `/public/reveal.js/css/reveal.css' not found.
[2017-04-19 07:52:59] ERROR `/public/reveal.js/css/theme/white.css' not found.
[2017-04-19 07:52:59] ERROR `/public/reveal.js/lib/css/zenburn.css' not found.
[2017-04-19 07:52:59] ERROR `/public/reveal.js/lib/js/head.min.js' not found.
[2017-04-19 07:52:59] ERROR `/public/reveal.js/js/reveal.js' not found.
```
and indeed, `/public/reveal.js` is  empty

**Comment by kohlhase** [comment]
I wonder if I should just `git subrepo` the repos https://github.com/hakimel/reveal.js into the right place.

**Comment by kohlhase** [comment]
OK, I have done that locally, and it seems to fix the problem. committing.

**Comment by nthiery** [comment]
>    OK, I have done that locally, and it seems to fix the problem.
>    committing.

Had you done a recursive checkout? I thought I had done so already.
Thanks for the fix if not!

**Comment by VivianePons** [comment]
This was a subrpo checking out directly from the reveal repo (but indeed it
needed a recursive checkout which should have been added to the readme or
something). Indeed, I am not sure your fix was actually a fix. When I try
to pull now, it gives me an error because the file you added are going to
override mine (which I checked from the subrepo). It might do the same on
the website and stopping it from updating...

2017-04-19 16:25 GMT+02:00 Nicolas M. Thiéry <notifications@github.com>:
